trees/redblacktree/redblacktree.go

```diff
@@ -31,7 +32,7 @@ const (
 type Tree struct {
 	Root       *Node
 	size       int
-	Comparator utils.Comparator
+	comparator utils.Comparator
```

Comment:
This is a mistake of mine from an earlier version where I had the new Tree interface have a getter for the comparator in order to make new trees. It can be undone.